content/en/community/hackathons/sessions/contributing-to-unikraft/content/Practical-Work_13.-Final-Verification.md

```diff
@@ -1 +1,3 @@
 Test the resulted library by calling the test function from the `app-kdtree` application.
+
+**Note**: Don't forget to include the updated library header in the main application file.
```

Comment:
Here, I'd give a snippet of how the output of a successful run of the app would look like.